### DISCUSSION TITLE
[documentation] Database Adapters - TypeORM table names

### DISCUSSION THREAD

**Discussion by darklightcode** [nextauthjs/next-auth#2055] [Help]
## Question 💬

I'm new to next-auth, so first thing i did i went to the documentation page, [the mysql page](https://next-auth.js.org/adapters/typeorm/mysql) more exactly, to set up the tables. Everything went ok, until the project started and `typeorm-legacy-adapter` created a new set of tables. The difference was that in the documentation the table names are pluralized and in the `typeorm-legacy-adapter` schema the names are not. 

Postgres and Microsoft SQL Server pages are also pluralized. Everything works fine, but regarding the pluralized names from the docs, was it meant to be like that or they're just typos ?

**Comment by balazsorban44** [comment]
So it's @iaincollins who created the typeorm adapter, but from the source code, if I think correctly, it always pluralizes by default:
https://github.com/nextauthjs/adapters/blob/canary/packages/typeorm-legacy/src/lib/naming-strategies.js

https://github.com/nextauthjs/adapters/blob/canary/packages/typeorm-legacy/src/lib/transform.js
